src/main/openapi/search-config-openapi-v1.yaml

```diff
@@ -0,0 +1,361 @@
+              $ref: '#/components/schemas/CreateSearchConfigRequest'
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreateSearchConfigResponse'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetailResponse'
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetailResponse'
+  /v1/searchConfig/search:
+    post:
+      tags:
+        - SearchConfigV1
+      summary: Finds search configs by criteria.
+      description: The search configs are found by search criteria.
+      operationId: findBySearchCriteria
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SearchConfigSearchRequest'
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetSearchConfigResponse'
+        "400":
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetailResponse'
+        "500":
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetailResponse'
+components:
+  schemas:
+    SearchConfig:
+      type: object
+      properties:
+        id:
+          type: string
+        page:
+          type: string
+        name:
+          type: string
+        modificationCount:
+          type: integer
+        application:
+          type: string
+        fieldListVersion:
+          type: integer
+          description: Version increment of the fields in the UI which you should use when you are making incompatible changes to those fields.
+        isReadOnly:
+          type: boolean
+          description: Defines whether this config can be changed in the UI
+        isAdvanced:
+          type: boolean
+          description: Indicates whether the advanced mode should be displayed
+        columns:
+          type: array
+          items:
+            type: string
+        values:
+          type: object
+          additionalProperties:
+            type: string
+    CreateSearchConfigRequest:
+      type: object
+      required:
+        - 'page'
+        - 'name'
+        - 'fieldListVersion'
+        - 'isReadonly'
+        - 'isAdvanced'
+        - 'columns'
+        - 'values'
+      properties:
+        page:
+          type: string
+        name:
+          type: string
+        application:
```

Comment:
move it to the path

Reply:
Please can you check this again and let me know where the comment applies and why?
Since i made many updates, the lines are outdated